openshift/Makefile

```diff
@@ -4,3 +4,11 @@ DIR := $(strip $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
 include $(addprefix $(DIR)/vendor/github.com/openshift/build-machinery-go/make/, \
     targets/openshift/deps.mk \
 )
+
+include $(DIR)/../.bingo/Variables.mk
+
+MANIFEST_DIR := $(DIR)/../manifests
+
+.PHONY: manifests
+manifests: $(KUSTOMIZE) $(YQ)
+	$(DIR)/generate-manifests.sh
```

Comment:
Perhaps fine for a follow-up, but do we need a `verify` target here like this:

```
verify: manifests vendor
	git diff --exit-code
```

So that we can call it in prow to make sure everything is up-to-date?